node/node.go

```diff
@@ -561,7 +561,15 @@ func (node *Node) Start() error {
 					)
 
 					if err != nil {
-						errChan <- withError{err, msg.GetFrom()}
+						// block the peer if it is not in the current committee
+						// FIXME: this is for testing only, will need to enable it after we increase the slot to 640
+						// as the internal nodes may contain invalid keys and we don't want to block internal nodes
+						if err == shard.ErrValidNotInCommittee {
```

Comment:
Why are we only blocking this specific error? Maybe we should also block invalid signature and more?

Reply:
Yes, we can also block other errors.  This is the initial intention as we have seen many error messages like this. Ofc, to protect the network, we shall also block on other errors.